src/components/views/home/Hero.js

```diff
@@ -50,6 +50,9 @@ const H1Title = styled.h1`
   color: ${(props) => props.theme.colors.linkGrey};
   font-size: 2.25rem;
   ${(props) => props.theme.mq.xlarge} {
+    font-size: 1.35rem;
```

Comment:
Je ne comprends toujours pas ce changement de size

Reply:
Pour laisser la place à l'image, sinon le texte prend trop de place. Je reconnais que le changement de taille du texte est inhabituel, mais ça convient à Valentin.

Reply:
Je laisse ouvert cette PR, pour une fois il y aura 2 versions de la même fonctionnalité, on verra laquelle convient la mieux à Clémence in fine.